projects/plugins/jetpack/_inc/client/search/dashboard/module-control.jsx

```diff
@@ -80,6 +85,34 @@ function Search( props ) {
 	const togglingInstantSearch = !! props.isSavingAnyOption( 'instant_search_enabled' );
 	const isSavingEitherOption = togglingModule || togglingInstantSearch;
 
+	const isInstantSearchCusomizeButtonDisabled =
```

Comment:
Should this be `isInstantSearchCustomizeButtonDisabled`?

Reply:
fixed in 658a9fa41